rs/sns/testing/dfx.json

```diff
@@ -0,0 +1,11 @@
+{
+  "canisters":
```

Comment:
Why do we need this file?

Reply:
The README in this crate asks to deploy the testing canister using `dfx` to run the scenario using the `ic-agent` backend. AFAIU, to be able to deploy canisters using `dfx`, `dfx.json` is required